homeassistant/components/sabnzbd/sensor.py

```diff
@@ -1,39 +1,120 @@
 """Support for monitoring an SABnzbd NZB client."""
 from __future__ import annotations
 
-from homeassistant.components.sensor import SensorEntity
-from homeassistant.core import HomeAssistant
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
-from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
-
-from . import (
-    DATA_SABNZBD,
-    SENSOR_TYPES,
-    SIGNAL_SABNZBD_UPDATED,
-    SabnzbdSensorEntityDescription,
+
+from . import DOMAIN, SIGNAL_SABNZBD_UPDATED, SabnzbdApiData
+from ...config_entries import ConfigEntry
```

Comment:
Please don't use relative imports for extra package imports. It's not compatible if this integration would be copied to a custom integration.

Use absolute imports for non package names.